packages/kit/src/runtime/server/validate-headers.js

```diff
@@ -0,0 +1,59 @@
+/** @type {Set<string>} */
+const VALID_CACHE_CONTROL_DIRECTIVES = new Set([
+	'max-age',
+	'public',
+	'private',
+	'no-cache',
+	'no-store',
+	'must-revalidate',
+	'proxy-revalidate',
+	's-maxage',
+	'immutable',
+	'stale-while-revalidate',
+	'stale-if-error',
+	'no-transform',
+	'only-if-cached',
+	'max-stale',
+	'min-fresh'
+]);
+
+const CONTENT_TYPE_PATTERN = /^[a-z]+\/[-+.\w]+$/i;
+
+/** @type {Record<string, (value: string) => void>} */
+const HEADER_VALIDATORS = {
+	'cache-control': (value) => {
+		const directives = value
+			.split(',')
+			.map((directive) => directive.trim().split('=')[0].toLowerCase());
```

Comment:
```suggestion
			.map((directive) => directive.trim().split('=').at(0)?.toLowerCase());
```

Is it possible for the string to end up with an empty entry after splitting on `','`? Like `cache-control: stale-while-revalidate,,no-transform` or something? Obviously that's still wrong, but hopefully we'd catch that and show an error, not throw a hard-to-debug runtime error :thinking:

Reply:
This was a good point! I've introduced an additional check to catch the empty directives.